src/Regions.ts

```diff
@@ -1,5 +1,5 @@
 export enum Regions {
-    NorthAmerica = "NorthAmerica",
-    RestOfWorld = "RestOfWorld",
-    China = "China"
+    NorthAmerica = "na",
```

Comment:
Revert these to older value, compatibility.

Reply:
Sure, I've changed it back and used a different method to get the region codes based on the region enum.